Commands/security/api-collection/readme.md

```diff
@@ -0,0 +1,17 @@
+# [Group] _security api-collection_
+
+Manage Azure API Management API connections to Microsoft Defender for APIs.
+
+## Commands
+
+- [create](/Commands/security/api-collection/_create.md)
+: Onboard an Azure API Management API to Microsoft Defender for APIs. The system will start monitoring the operations within the Azure Management API for intrusive behaviors and provide alerts for attacks that have been detected.
+
+- [delete](/Commands/security/api-collection/_delete.md)
```

Comment:
Can we use a different command than "delete", like "offboard-api-mgmt-api"? "delete" is too generic and will not account for future scenarios where we may want to offboard apiCollections from other resources like App Services or K8s.

Reply:
Or even just "offboard"